manifests/p/pnpm/pnpm/7/7.9.0/pnpm.pnpm.7.locale.en-US.yaml

```
@@ -0,0 +1,42 @@
+# Created with YamlCreate.ps1 v2.1.3 $debug=NVS1.7-2-5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: pnpm.pnpm.7
+PackageVersion: 7.9.0
+PackageLocale: en-US
+Publisher: pnpm
+PublisherUrl: https://github.com/pnpm/pnpm
+PublisherSupportUrl: https://github.com/pnpm/pnpm/issues
+# PrivacyUrl: 
+Author: pnpm contributors
+PackageName: pnpm 7
+PackageUrl: https://pnpm.io/
+License: MIT License
+LicenseUrl: https://github.com/pnpm/pnpm/blob/main/LICENSE
+Copyright: |-
+  Copyright (c) 2015-2016 Rico Sta. Cruz and other contributors
+  Copyright (c) 2016-2022 Zoltan Kochan and other contributors
```

```suggestion
Copyright: Copyright (c) 2016-2022 Zoltan Kochan and other contributors
```
It's best to only include the active copyright, not the expired one